### DISCUSSION TITLE
Error installing a module with local dependencies in Docker

### DISCUSSION THREAD

**Discussion by zhimbura** [npm/feedback#253] [CLI] [answered]
To reproduce the bug, I published the module https://www.npmjs.com/package/npm-module-for-example-bug-xx

### It installs locally without problems 
![image](https://user-images.githubusercontent.com/15249713/109520142-f6eb7b00-7abc-11eb-8884-e1a4d02d8af2.png)

### But when installing in docker I get the error
![image](https://user-images.githubusercontent.com/15249713/109520232-0d91d200-7abd-11eb-9cb3-d4ddcb352e6c.png)

```dockerfile
FROM node:14.16.0

RUN npm install npm-module-for-example-bug-xx

RUN echo "ok"
```

**Comment by zhimbura** [comment]
How fix it?

**Reply by MylesBorins** [reply]
hey @Tihon-Ustinov would you be able to open an issue over at https://github.com/npm/cli

This discussions form is focused on feature requests not bug reports

Thanks!

**Comment by zhimbura** [comment, marked as answer]
Upgrage to npm  7.5 fix it
